### PR TITLE
Clarity Edit

### PR DESCRIPTION
**Pull request by ucheNkadiCode**
(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@ucheNkadiCode : Thanks for your contribution! The author, @John-Hart, has been notified to review your proposed change.

File: docs/vsto/walkthrough-designing-an-outlook-form-region.md [hunk 1]
```diff
@@ -155,6 +155,8 @@ ms.workload:
 
 5.  Reopen the **Ann Beebe** contact item.
 
+    In Outlook 365, this can be done in the **Find** group by either opening the Address Book for Contacts or typing Ann Beebe into **Search People**.
```
Comment (John-Hart) [inline docs/vsto/walkthrough-designing-an-outlook-form-region.md]: I would remove "365", as that would imply it doesn't work in the desktop version

**Comment by PRMerger17** [comment]
@ucheNkadiCode : Thanks for your contribution! The author, @John-Hart, has been notified to review your proposed change.

File: docs/vsto/walkthrough-designing-an-outlook-form-region.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ ms.workload:
 
 - [!INCLUDE[vsto_vsprereq](../vsto/includes/vsto-vsprereq-md.md)]
 
-- [!INCLUDE[Outlook_15_short](../vsto/includes/outlook-15-short-md.md)] or [!INCLUDE[Outlook_14_short](../vsto/includes/outlook-14-short-md.md)].
+- [!INCLUDE[Outlook_14_short](../vsto/includes/outlook-14-short-md.md)] or greater.
```
Comment (John-Hart) [inline docs/vsto/walkthrough-designing-an-outlook-form-region.md]: I would suggest or "newer" instead of greater

**Comment by PRMerger19** [comment]
@ucheNkadiCode : Thanks for your contribution! The author, @John-Hart, has been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@John-Hart - i'm taking your approval for sign off and merging this PR.

@ucheNkadiCode - can you submit changes through the private repo in the future? here's some information about that:

Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.  

Read about [authors working in the public repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master) for more information.  

If you need help getting set up to work in the private repo, see [Set up Git repository locally for documentation](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-local?branch=master). 

Thanks!

**Pull request merged**